packages/launchpad/cypress/e2e/config-files-error-handling.cy.ts

```diff
@@ -147,8 +147,31 @@ describe('Launchpad: Error System Tests', () => {
     cy.visitLaunchpad()
     cy.contains('h1', cy.i18n.launchpadErrors.generic.configErrorTitle)
     cy.percySnapshot()
+
+    cy.withCtx(async (ctx) => {
+      await ctx.actions.file.writeFileInProject('cypress.config.ts', 'export default { e2e: { supportFile: false } }')
+    })
+
+    cy.findByRole('button', { name: 'Try again' }).click()
+
+    cy.get('h1').should('contain', 'Welcome to Cypress')
+  })
+
+  it(`clears the error correctly after first 'try again' attempt`, () => {
+    cy.scaffoldProject('config-with-ts-syntax-error')
+    cy.openProject('config-with-ts-syntax-error')
+    cy.visitLaunchpad()
+    cy.contains('h1', cy.i18n.launchpadErrors.generic.configErrorTitle)
+    cy.percySnapshot()
```

Comment:
```suggestion
    cy.contains('h1', cy.i18n.launchpadErrors.generic.configErrorTitle)
```
We can remove this percySnapshot, it's not covering any more than the test above.